scripts/ci.genycloud-login.sh

```diff
@@ -1,5 +0,0 @@
-#!/bin/bash -ex
-
-which gmsaas
-gmsaas --version
-gmsaas auth login "$GENYCLOUD_USERNAME" "$GENYCLOUD_PASSWORD"
```

Comment:
So I guess we can also remove these secrets from Buildkite, right?

Reply:
correct. Will do it soon